engines/sci/engine/workarounds.cpp

```diff
@@ -342,6 +342,8 @@ const SciWorkaroundEntry uninitializedReadWorkarounds[] = {
 	{ GID_LSL6HIRES,      -1,    85,  0,             "LL6Inv", "init",                            NULL,     0, { WORKAROUND_FAKE,   0 } }, // when creating a new game
 	{ GID_LSL6HIRES,      -1, 64950,  1,            "Feature", "handleEvent",                     NULL,     0, { WORKAROUND_FAKE,   0 } }, // at least when entering swimming pool area
 	{ GID_LSL6HIRES,      -1, 64964,  0,              "DPath", "init",                            NULL,     1, { WORKAROUND_FAKE,   0 } }, // during the game
+	{ GID_LSL7,           50, 64017,  0,             "oFlags", "clear",                           NULL,     0, { WORKAROUND_FAKE,   0 } }, // when starting the demo
+	{ GID_LSL7,         9000, 64892,  0,      "oEventHandler", "killAllEventHogs",                NULL,     1, { WORKAROUND_FAKE,   0 } }, // when clicking on help in the dice mini game
```

Comment:
I can't reproduce the problem with "how to play" anymore. Also, the oFlags workaround needs to be applied to all rooms - it's reproducible whenever the pop up menu is clicked to jump to the mini-game. I'll adapt this accordingly and submit to master